cmake/emu/qemu.cmake

```
@@ -267,8 +267,8 @@ if(CONFIG_X86_64)
     COMMAND
     ${CMAKE_OBJCOPY}
     -R .locore
-    ${CMAKE_BINARY_DIR}/zephyr-qemu.elf
-    ${CMAKE_BINARY_DIR}/zephyr-qemu-main.elf
+    ${CMAKE_BINARY_DIR}/zephyr/zephyr-qemu.elf
+    ${CMAKE_BINARY_DIR}/zephyr/zephyr-qemu-main.elf
```

We have `ZEPHYR_BINARY_DIR` for this purpose.
```suggestion
    ${ZEPHYR_BINARY_DIR}/zephyr-qemu-main.elf
```